CODEOWNERS

```diff
@@ -81,12 +81,14 @@
 
 /pkg/packetbrokeragent @johanstokking @adriansmares
 
-/pkg/devicetemplateconverter @adriansmares @neoaggelos
-/pkg/devicetemplates @adriansmares @neoaggelos
-/pkg/qrcode @adriansmares @neoaggelos
-/pkg/qrcodegenerator @adriansmares @neoaggelos
+/pkg/devicetemplateconverter @adriansmares @KrishnaIyer
+/pkg/devicetemplates @adriansmares @KrishnaIyer
+/pkg/qrcode @adriansmares @KrishnaIyer
+/pkg/qrcodegenerator @adriansmares @KrishnaIyer
 
-/pkg/devicerepository @adriansmares @neoaggelos
+/pkg/deviceclaimingserver @johanstokking @KrishnaIyer
```

Comment:
Please replace me by @nicholaspcr 
If there's review necessary you can always request but let's not block on me